src/components/EntityInfo/EntityMenu.vue

```diff
@@ -352,13 +387,18 @@ export default class EntityMenu extends Mixins(AllowableActionsMixin) {
   promptDissolve (): void {
     if (!this.isGoodStanding) {
       this.emitNotInGoodStanding(NigsMessage.DISSOLVE)
-      return
+    } else {
+      this.emitConfirmDissolution()
     }
-    this.emitConfirmDissolution()
   }
 
   goToAmalgamationSelection (): void {
-    this.$router.push({ name: Routes.AMALGAMATION_SELECTION, params: { filingId: '0' } })
+    this.$router.push({ name: Routes.AMALGAMATION_SELECTION })
```

Comment:
This route/view doesn't need "id=0" protection, which is used elsewhere so user cannot go right to the filing URL. Here, we don't need to protect against see.

See also small change to Amalgamation Selection view.